src/registrar/fixtures.py

```diff
@@ -0,0 +1,256 @@
+    """
+
+    ADMINS = [
+        {
+            "username": "c4a0e101-73b4-4d7d-9e5e-7f19a726a0fa",
+            "first_name": "Seamus",
+            "last_name": "Johnston",
+        },
+        {
+            "username": "d4c3bd84-dc3a-48bc-a3c3-f53111df2ec6",
+            "first_name": "Igor",
+            "last_name": "",
+        },
+        {
+            "username": "ee80bfe0-49ad-456d-8d82-e2b608a66517",
+            "first_name": "Logan",
+            "last_name": "",
+        },
+    ]
+
+    @classmethod
+    def load(cls):
+        logger.info("Going to load %s users" % str(len(cls.ADMINS)))
+        for admin in cls.ADMINS:
+            try:
+                user, _ = User.objects.get_or_create(
+                    username=admin["username"],
+                )
+                user.is_superuser = True
+                user.first_name = admin["first_name"]
+                user.last_name = admin["last_name"]
+                user.is_staff = True
+                user.is_active = True
+                user.save()
+                logger.debug("User object created for %s" % admin["first_name"])
+                UserProfile.objects.get_or_create(user=user)
+                logger.debug("Profile object created for %s" % admin["first_name"])
+            except Exception as e:
+                logger.warning(e)
+        logger.debug("All users loaded.")
+
+
+class DomainApplicationFixture:
+    """
+    Load domain applications into the database.
+
+    Make sure this class' `load` method is called from `handle`
+    in management/commands/load.py, then use `./manage.py load`
+    to run this code.
+    """
+
+    # any fields not specified here will be filled in with fake data or defaults
+    # NOTE BENE: each fixture must have `organization_name` for uniqueness!
+    DA = [
+        {
+            "status": "started",
+            "organization_name": "Example - Finished but not Submitted",
+        },
+        {
+            "status": "started",
+            "organization_name": "Example - Just started",
+            "organization_type": "federal",
+            "federal_agency": None,
+            "federal_type": None,
+            "address_line1": None,
+            "address_line2": None,
+            "city": None,
+            "state_territory": None,
+            "zipcode": None,
+            "urbanization": None,
+            "purpose": None,
+            "security_email": None,
+            "anything_else": None,
+            "is_policy_acknowledged": None,
+            "authorizing_official": None,
+            "submitter": None,
+            "other_contacts": [],
+            "current_websites": [],
+            "alternative_domains": [],
+        },
+        {
+            "status": "submitted",
+            "organization_name": "Example - Submitted but pending Investigation",
+        },
+        {
+            "status": "investigating",
+            "organization_name": "Example - In Investigation",
+        },
+    ]
```

Comment:
These are a little funky when loaded for me - the fake data doesn't seem to be loaded when I go to the form.